ru/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -147,7 +147,7 @@
 <!ENTITY plainTextFormatCmd.accesskey "P">
 <!ENTITY htmlFormatCmd.label "Форматированный текст (HTML)">
 <!ENTITY htmlFormatCmd.accesskey "H">
-<!ENTITY bothFormatCmd.label "Обычный текст и HTML">
+<!ENTITY bothFormatCmd.label "Простой текст и HTML">
```

Comment:
CORRECTIONS NEEDED FOR LINE 115:
let's stick to original version, since it is somewhat more common
+<!ENTITY bothFormatCmd.label "Обычный текст и HTML">